gerrytools/data/census.py

```diff
@@ -42,25 +44,39 @@ def census10(state, columns={}, geometry="block"):
         A DataFrame with columns renamed according to their Census description
         designation and a unique identifier column for joining to geometries.
     """
+    # Check whether the geometry is right. If not, warn the user and set it
+    # properly.
+    if geometry not in {"block", "tract", "block group"}:
+        print(f"Geometry \"{geometry}\" not accepted; defaulting to \"block\".")
+        geometry = "block"
```

Comment:
This could be easy to miss. I'd suggest raising a `ValueError` here instead.
```suggestion
    if geometry not in {"block", "tract", "block group"}:
        raise ValueError(f'Geometry "{geometry}" not accepted.')
```